tasks/client_config.yaml

```diff
@@ -1,12 +1,24 @@
 ---
+- name: Check for root password
+  stat:
+    path: /usr/local/etc/mysql-password
+  register: mysql_password
```

Comment:
Please use `mariadb_` prefix for `mariadb` role.

Reply:
ok.

Reply:
Fixed with 347cb9ac5dc5cb1a677646e0156dff326492becc